docroot/themes/contrib/civictheme/templates/views/views-view.html.twig

```diff
@@ -35,29 +35,6 @@
 
 {% extends '@organisms/listing/listing.twig' %}
```

Comment:
We use the component as is - it has everything we need now with added title and link.